### PR TITLE
add distributed solar for Australia

### LINKED ISSUE #511: Add Australian small scale solar
Small scale solar is a significantly large proportion of solar in australia.

Details on installed capacity can be found here

https://d284f79vx7w9nf.cloudfront.net/attachments/analysis/7/states.csv

from http://pv-map.apvi.org.au/historical#4/-26.67/134.12

production data can be found here

http://pv-map.apvi.org.au/live

### PR DESCRIPTION
**Pull request by jarek**
This will fix #511.

Testing notes:

It is currently winter in Australia. Sun sets relatively early and production is relatively low. Eastern states are on UTC+10, South Australia UTC+9:30, Western Australia UTC+8.

Code notes:

There is one source for both eastern states and Western Australia. Because those are currently handled by separate parsers, I put the solar parser in a separate module that is included from both parsers. That means that the parsers can no longer be run with `python AU.py` (due to "Attempted relative import in non-package"), so I added instructions on how to run it from command-line.

An alternative would be to merge AU.py and AU_WA.py into one file, then AU_solar.py can also be merged into it. Then we can run `python AU.py` to test in command line. Let me know if you would prefer that.

I modified the WA parser to return 24 hours of history rather than the most recent data point, since it's available in the source.

Also included minor cleanups and made both parsers compatible with both Python 2 and Python 3. This was mostly just putting brackets around `print()` calls.

`'Macadamia Nut Shells'` key was duplicated in `AMEO_CATEGORY_DICTIONARY` in AU.py.

### CODE REVIEW

File: parsers/AU.py [hunk 1]
```diff
@@ -4,8 +4,18 @@
 import requests
 
 import json
+import numpy as np
 import pandas as pd
-import math
+
+try:
```
Comment (corradio) [inline parsers/AU.py]: Hmm this sounds overkill.
Maybe we can add the PYTHONPATH to the pylint command here:
https://github.com/tmrowco/electricitymap/blob/master/circle.yml#L20

File: parsers/AU.py [hunk 2]
```diff
@@ -321,6 +330,7 @@ def fetch_production(country_code=None, session=None):
     df = pd.read_csv(url)
     data = {
         'countryCode': country_code,
+        'datetime': arrow.get('2001-01-01T00:00:00+1000').datetime,
```
Comment (corradio) [inline parsers/AU.py]: Seems dangerous to hardcode data here?

**Comment by jarek** [comment]
Good idea. Moving AU_solar.py into a lib subdirectory seems to fix it, at least for running in command-line and for pylint. Can you see if it'll work on the data collection setup?

File: parsers/AU.py (hunk 2 above)
Comment (jarek) [inline parsers/AU.py]: It'd never get through unless there was no data in the upstream API at all, and at that point your backend would probably ignore it anyway? But true, I was trying to be a bit too clever, rewrote it.

**Pull request merged**